README.md

```diff
@@ -47,3 +53,48 @@ public protocol ZippyVCDelegate: class {
     func onCompletedWithError(error: ZippyError)         // Called when an error (server problems, code errors) has occured during process
 }
 ```
+
+### Results
+
+To receive the user's results, create a `ZippyVCDelegate` extension using methods `onCompletedSuccessfully(result: ZippyResult)` and `onCompletedWithError(error: ZippyError)`
+
+```Swift
+extension ViewController: ZippyVCDelegate {
+    func onCompletedSuccessfully(result: ZippyResult) {
+        print(result)
+    }
+
+    func onCompletedWithError(error: ZippyError) {
+        print(error.localizedDescription)
+        ...
+    }
+}
+```
+
+If the session was successful `onCompletedSuccessfully` is called which contains the sessions result in `result`
+
+If there was an error `onCompletedWithError` is called which contains the error message in `error`
+
+### Optional callback
+
+If you want you can use 3 additional functions by adding a `ZippyCallback` extension
+
+```Swift
+extension ViewController: ZippyCallback {
+    func onSubmit() {
+        // called when all images are sent
+    }
+    func onTextExtracted() {
+        // called when API receives a result
```

Comment:
Te neprasās parametrā jau to extracted informāciju?

Reply:
Es nezinu, nesaprotu laikam. Kristaps teica, ka iedošot tam callbackam endpointu, bet nav vēl iedevis.

Reply:
OK